javascript/ql/src/semmle/javascript/security/internal/SensitiveDataHeuristics.qll

```diff
@@ -113,8 +114,9 @@ module HeuristicNames {
 
   /**
    * Holds if `name` may indicate the presence of sensitive data, and
-   * `name` does not indicate the presence of data that is hashed or encrypted, which would have
-   * rendered the data non-sensitive. `classification` describes the kind of sensitive data involved.
+   * `name` does not indicate that the data is in fact non-sensitive (for example since
+   * it is hashed or encrypted). `classification` describes the kind of sensitive data
+   * involved.
    *
    * That is, one of the rexeps from `maybeSensitiveRegexp` matches `name` (with the
    * given classification), and none of the regexps from `notSensitiveRegexp` matches
```

Comment:
just a typo, will fix up :+1:

Reply:
It would probably be more mainstream to use regexes... but I wanted to match the wording we use in CodeQL, which is `regexp`, for example in `regexpMatch`